### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by Devarora-0981** [community/community#35619] [Other Feature Feedback, Questions, & Ideas] [answered]
Today When I tried to log in into my account @Devarora0981 Then I saw that my account was suspended. Can I get a proper reason behind it? Because I lost all my codes and repositories.

**Comment by radar-blip** [comment, marked as answer]
For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Reply by ghost** [reply]
Same Here. They Shouldnt have done this without any warnings or Notice.

**Reply by zcrew0x** [reply]
me too

**Reply by Abdul48608417** [reply]
Me too

**Reply by dungng1321** [reply]
Me too .Did you get your account back?

**Reply by ghost** [reply]
Same. Did you manage to solve it. I have been waiting for a response from GitHub support for 6 weeks now with no response. I only received the automatic email saying "We are experiencing high support volumes, there may be longer wait times". I honestly doubt GitHub has about 6 weeks worth of tickets.

**Reply by papaimistry2** [reply]
@zcrew0x bro how could you got your account back?
my account got also suspended plz help , my user id is @papaimistry

**Reply by papaimistry2** [reply]
@zcrew0x  bro how could you got your account back?
 My account is also suspended. Please help me to unsuspend this account. My user id @papaimistry

**Reply by imohamed-rizwan** [reply]
same here ... one of my account got suspended just now without any prior information's.
does anyone know how long it will take to get a response from GitHub team with actual reason for suspension.

**Reply by taciaraf** [reply]
Account Name: taciara https://github.com/taciara

My account has been suspended for 1 week, I have important projects in my repository. I really need to access my account again. How can we solve this serious problem?

**Reply by anishDevMage** [reply]
Did you all get your account's unsuspended?
Even my previous account got suspended https://github.com/anishDev419
so created a new one https://github.com/anishDevMage

even this one receives the suspended error when i try to push to the remote branches

help.

**Reply by ghost** [reply]
did you get your account back?
i recently got suspended too
can you tell me how to get my account back please.

**Reply by imohamed-rizwan** [reply]
@mdpstory How long has it been since your account was suspended? Please send an email to the support team (I hope they will respond to you) and wait a few working days as the ban may be lifted automatically if you did not violate the rules and policies.

**Reply by ghost** [reply]
![Screenshot from 2024-04-16 18-07-33](https://github.com/community/community/assets/88600423/e44ce4af-8c11-4883-8369-7ee04c327197)

i already submitted reinstatement request, its been 3 weeks with no answer.
i really dont know what i do to get suspended tho.

**Reply by DebiduttaPanigrahi** [reply]
My account DUTTA-78150 encountered a similar problem, being suspended for breaching terms and conditions. I'm concerned about the timeframe for resolution, given that both my personal and collaborative projects rely on it. Although I've submitted a ticket, I'm still awaiting a response. Please prioritize this matter for rectification.

**Reply by ghost** [reply]
@mdpstory feel you bro, this account suspension messed up my entire life

**Reply by DebiduttaPanigrahi** [reply]
My account was banned without any reason. After I raised a ticket, it was resolved, but from that day, my account has not been visible to others. I have tried everything, but when someone tries to view my account, it shows a 404 error. I have raised a ticket again, but the issue remains the same. Please help me with this; it is my main GitHub account.

**Reply by edujeetX** [reply]
> Account Name: taciara https://github.com/taciara
> 
> My account has been suspended for 1 week, I have important projects in my repository. I really need to access my account again. How can we solve this serious problem?

I see your account is recovered successfully, could you help me with that. My account got suspended too this morning.

**Comment by raviisharmaa** [comment]
![MicrosoftTeams-image](https://user-images.githubusercontent.com/115176138/197150595-021a4e72-ce81-48eb-9bcc-98a531a64409.png)

**Reply by DebiduttaPanigrahi** [reply]
My account DUTTA-78150 encountered a similar problem, being suspended for breaching terms and conditions. I'm concerned about the timeframe for resolution, given that both my personal and collaborative projects rely on it. Although I've submitted a ticket, I'm still awaiting a response. Please prioritize this matter for rectification.

**Comment by raviisharmaa** [comment]
Why did you suspended my account ? How do I get rid of this ?

**Comment by JasonLinnnnnn** [comment]
hi there, do you get your account back, i suffered the same issue, too...

**Reply by raviisharmaa** [reply]
Hi , do one fix , generate new ssh key for your github.

**Reply by DebiduttaPanigrahi** [reply]
My account DUTTA-78150 encountered a similar problem, being suspended for breaching terms and conditions. I'm concerned about the timeframe for resolution, given that both my personal and collaborative projects rely on it. Although I've submitted a ticket, I'm still awaiting a response. Please prioritize this matter for rectification.

**Reply by valtriac1** [reply]
same

**Comment by Haridas0907** [comment]
remote: Your account is suspended. Please visit https://support.github.com for more information.
fatal: unable to access 'https://github.com/HaridasChavan/Tingo-Master.git/': The requested URL returned error: 403

**Reply by DebiduttaPanigrahi** [reply]
My account DUTTA-78150 encountered a similar problem, being suspended for breaching terms and conditions. I'm concerned about the timeframe for resolution, given that both my personal and collaborative projects rely on it. Although I've submitted a ticket, I'm still awaiting a response. Please prioritize this matter for rectification.

**Comment by imohamed-rizwan** [comment]
Don't know what happened, after few days from the incident the GitHub team sent me a mail which says it was a mistake from their side and my account can access from the moment. so for my case it was automatically unlocked after 3-4 days.

**Reply by DebiduttaPanigrahi** [reply]
My account DUTTA-78150 encountered a similar problem, being suspended for breaching terms and conditions. I'm concerned about the timeframe for resolution, given that both my personal and collaborative projects rely on it. Although I've submitted a ticket, I'm still awaiting a response. Please prioritize this matter for rectification.
WHAT TO DO PLEASE

**Comment by anishDevMage** [comment]
My account was suspended due to detection of malware in the system I was
using the github account on.
I did enquire for assistance and they have sent me response regarding this,
telling me to ask for assistance in a generated ticket link they have
provided.
However, currently I am trying to manually retrieve my data from the said
system and create a safe environment before moving on to reinstatement of
the account.

**Reply by DebiduttaPanigrahi** [reply]
TO WHOM I CAN MAIL GIVE ME THE MAIL ID PLEASE

**Comment by anishDevMage** [comment]
The mailing ID I tried on was ***@***.***
However, they might have blocked any incoming mail.

So I advice you search for github support page

In the webpage you have to search for ticket generation in order to enquire
you issue

They will see to it from there.

**Comment by ghost** [comment]
@Devarora-0981 
Did you get a response? Mine got suspended too, it has been more than a month and still no response.
ticket: #2719793

**Reply by ghost** [reply]
nah, almost 2 month
I think i got suspended because i have 2 github account, what about you?

**Reply by ghost** [reply]
Got hacked, all because my account was using a old email of mine that got leaked, didn't had 2FA active since it didn't even existed when i created it.

**Reply by ghost** [reply]
Exactly the same thing happened to me, my email was filtered and they accessed the account, changing the emails. Luckily, I reported it so they could suspend it, but now since I sent the tickets they haven't given me a response.
tickets: #2818991

**Reply by DebiduttaPanigrahi** [reply]
My account was banned without any reason. After I raised a ticket, it was resolved, but from that day, my account has not been visible to others. I have tried everything, but when someone tries to view my account, it shows a 404 error. I have raised a ticket again, but the issue remains the same. Please help me with this; it is my main GitHub account.

**Reply by guebli-djillali-2** [reply]
i have the same issue, i have accout that was suspended today amd i am trying to find a solution

**Comment by ghost** [comment]
remote: Your account is suspended. Please visit https://support.github.com for more information.
fatal: unable to access 'https://github.com/SadiqArsalan/stack_overflow-next14.git/': The requested URL returned error: 403

**Reply by ghost** [reply]
Bro follow these steps you can solve that problem. please share your account suspended issue in screenshot form.

**Comment by guebli-djillali-2** [comment]
But i want to get it back, can you please help me to with that ?
________________________________

**Comment by guebli-djillali-2** [comment]
Hello, please can you tell me which steps I should follow ?

And about the screenashot, here is it :

This when I try to reset my password => So it says : Account suspended

My gmail is : ***@***.***

[cid:e113c956-bd92-491b-94e3-e55a1bf9c173]

[cid:eb6581e2-ec81-4cd7-beb9-44a7740336b0]
________________________________

**Comment by Priyanshux085** [comment]
*Subject:* Appeal for Account Reinstatement

*Message:*

Hello GitHub Support Team,

My name is Priyanshu, and my GitHub username is Priyanshu085. I am writing to request an appeal and reinstatement of my GitHub account, which has been banned due to a violation.

Recently, I was completing a form related to the GitHub Student Developer Pack and got banned suddenly, without any warning or clear explanation of the specific violation. I rely heavily on my GitHub account for hosting and managing my academic and personal projects. Being unable to access my account is significantly impacting my development work and learning.

I understand the importance of adhering to GitHub’s Terms of Service and Community Guidelines. If there were any inadvertent violations on my part, I sincerely apologize and am committed to ensuring full compliance in the future.

Please review my case and consider reinstating my account. If further information is needed, I am more than willing to provide it.

Thank you for your understanding and assistance.

Best regards,  
Priyanshu  
GitHub Username: Priyanshu085  
Email: abpriyanshu007@gmail.com

**Comment by lightla** [comment]
I want my code, if not recover my account, please give me code @github-staff 

account: light-souler
